### ISSUE TITLE
Cannot close eyedropper via keyboard

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/web-stories-wp#11051]
## Bug Description

While looking at #11049, I noticed that when the eyedropper is open, I cannot close it / cancel the action using the ESC key, which I thought would be a common thing.

Instead, it just removes the current element selection.

## Expected Behaviour

The element should stay selected, but the eyedropper should be cancelled

## Screenshots

I am pressing ESC a few times in this video. You can observe the "Saved Styles" section on the left to see when the element is being unselected.

https://user-images.githubusercontent.com/841956/159719848-81e96b31-f068-4ba0-b84b-937feec8a1f4.mov

## Additional Context

- Plugin Version: 1.19.0-alpha
- WordPress Version: 5.9
- Operating System: macOS 
- Browser: Chrome
